src/cellfunction/meshdata.jl

```diff
@@ -27,11 +28,21 @@ set_values!(data::MeshData, values::Union{Number, AbstractVector}) = data.values
 get_location(::MeshData{L}) where {L} = L()
 
 function LazyOperators.materialize(data::MeshData{CellData}, cInfo::CellInfo)
-    PhysicalFunction(x -> _make_callable(get_values(data)[cellindex(cInfo)], x))
+    value = get_values(data)[cellindex(cInfo)]
+    return wrap_value(value)
 end
 
-_make_callable(f, x) = f(x)
-_make_callable(a::Union{Number, AbstractArray}, x) = a
+function LazyOperators.materialize(
+    data::MeshData{FaceData},
+    side::AbstractSide{Nothing, <:Tuple{<:FaceInfo}},
+)
+    fInfo = get_args(side)[1]
+    return get_values(data)[faceindex(fInfo)]
+end
+
+wrap_value(value) = value
+wrap_value(value::Union{Number, AbstractArray}) = ReferenceFunction(ξ -> value)
```

Comment:
Is this still necessary?

Reply:
yes it is. See this MWE where, without this `wrap_value`, the example fails (whereas the first sucesses):
```julia
using Bcube
using StaticArrays
using LinearAlgebra

struct FakeMeshData{L <: Bcube.AbstractMeshDataLocation, T <: AbstractVector} <:
       Bcube.AbstractLazy
    values::T
end
function FakeMeshData(location::Bcube.AbstractMeshDataLocation, values::AbstractVector)
    FakeMeshData{typeof(location), typeof(values)}(values)
end
get_values(data::FakeMeshData) = data.values

function Bcube.materialize(data::FakeMeshData{Bcube.CellData}, cInfo::Bcube.CellInfo)
    return get_values(data)[Bcube.cellindex(cInfo)]
end

FakeMeshCellData(values::AbstractVector) = FakeMeshData(Bcube.CellData(), values)

#--- Test 1
n = 3
mesh = line_mesh(n)
dΩ = Measure(CellDomain(mesh), 2)

# tensor data
# data = [[1,2,3,4], [5,6,7,8], ...]
data = collect(reinterpret(SVector{4, Int}, collect(1:(4 * ncells(mesh)))))
d = Bcube.MeshCellData(data)
@show Bcube.compute(∫(d ⋅ [0, 1, 0, 0])dΩ) == [1.0, 3.0]

d = FakeMeshCellData(data)
@show Bcube.compute(∫(d ⋅ [0, 1, 0, 0])dΩ) == [1.0, 3.0]
```